data/migrations/20190521132221_trips.js

```diff
@@ -0,0 +1,24 @@
+
+exports.up = function (knex, Promise) {
+    return knex.schema.createTable('trips', trips => {
+      trips.increments();
+      // foreign key
+      trips
+        .integer('user_id')
+        .unsigned()
+        .notNullable()
+        .references('id')
+        .inTable('users');
+      trips.string('trip_type', 128).notNullable();
+      trips.string('title', 128);
+      trips.string('location', 128).notNullable();
+      trips.string('duration', 128);
+      trips.text('description');
+      trips.boolean('professional').defaultTo(false);
```

Comment:
Be aware of the quirks of sqlite3 databases; I think this column will still default to `null` instead of `false`.